content/resources/use-cases/sdm/index.md

```diff
@@ -0,0 +1,217 @@
+strenua), and Southern Boobook (Ninox boobook). All data, including audio
+recordings, annotations, and associated sensor metadata, is available for
+download and use from the [A2O data portal](https://acousticobservatory.org/data/find-data/). A GitHub repository
+detailing the complete methods for preparing the Hoot Detective data, prediction
+layers, and running preliminary SDM analyses with R is
+[also provided](https://github.com/andrew-1234/sdm-usecase-master).
+
+### Data preparation
+
+The Hoot Detective project resulted in 28723 segments of annotated audio, from
+37 sensors. Each segment of audio was 10 seconds in length, resulting in a total
+of just under 80 hours of annotated audio. Audio segments came from most days
+spanning between June, 2019, until July, 2021 (Figure 2). These annotations
+were summarised in R to get occurrence data for each species, which included
+determining if at any point the target species was recorded (presence) or was
+absent from recordings (absence).
+
+{{% figure src="./hoot_segments_per_day.png" caption="Figure 2: Histogram demonstrating total frequency of annotated audio segments per day in the Hoot Detective data set." width="100%"%}}
+
+A Powerful Owl presence was recorded in Western Australia, which is quite far
+outside of the recognised distribution. Therefore, Powerful Owl records were
+filtered further, using a buffered boundary of the known distribution extent
+derived from ALA records, to remove these potentially anomalous records. 
+
+The A2O uses a paired sensor design, with four sensors per site. The distance
+between sensors is typically within 500m to 5000m. Therefore, we first attempted
+to use environmental layers at a high spatial resolution (25m), to capture fine
+scale variation between proximate sensors.This came at a high computer
+processing cost. Even working with buffered extents, and using a Big Data PC,
+the process was slow, highlighting the challenges of using such high resolution
+data. As a result, we decided to instead use a 250m resolution for preliminary
+analyses in R, and a 1000m resolution on the ecocommons platform. To get a
+raster stack for the extent of Australia, the terra package in R was used to
+crop and mask layers to a vector of the Australian boundary. Layers were then
+resampled from their original resolution where necessary to 250m, using bilinear
+interpolation for continuous bioclim data, mean for forest canopy and height,
+and nearest neighbour for categorical data. All layers were projected to the
+project CRS (EPSG: 4326). For Powerful Owl, the raster stack was cropped and
+masked to the buffered boundary extent representing the known distribution. 
+
+### Analysis
+
+Powerful Owl presence data were used to extract environmental variable values
+from eastern Australia, and the resulting presence data and environmental
+rasters were used in a Maxent model in R [(see code)](https://github.com/andrew-1234/sdm-usecase-master).
```

Comment:
maybe you could link to a specific part of the repo that is more relevant to this sentence.